### PR TITLE
Inline `InflatableModel` class

### PR DESCRIPTION
**Pull request by tijmenb**
This inlines the code from the `InflatableModel` class, which is only used once.

It reduces the number of concepts in Whitehall, which can’t be a bad thing.

### CODE REVIEW

File: test/unit/inflatable_model_test.rb [hunk 1]
```diff
@@ -1,11 +0,0 @@
-require 'test_helper'
-
-class InflatableModelTest < ActiveSupport::TestCase
-  test "#initialize should assign to declared attributes from passed in hash" do
-    class InflatableModelTestClass < InflatableModel
-      attr_accessor :foo
-    end
-
-    assert_equal 'bar', InflatableModelTestClass.new(foo: 'bar').foo
```
Comment (tvararu) [inline test/unit/inflatable_model_test.rb]: Should this test move to `test/unit/models/frontend/statistics_announcement_test.rb`?

**Comment by tvararu** [comment]
A 👍  from me for reducing the mental overhead in Whitehall, one class at a time.

File: test/unit/inflatable_model_test.rb (hunk 1 above)
Comment (tijmenb) [inline test/unit/inflatable_model_test.rb]: I'm pretty sure that `StatisticsAnnouncementTest` would fail if this were wrong.

**Comment by gpeng** [comment]
:+1: when 🍏

**Comment by boffbowsh** [comment]
Is this code even needed in Whitehall at all any more? /cc @fofr

**Comment by tijmenb** [comment]
@boffbowsh it's used by the index page to present rummager documents. https://github.com/alphagov/whitehall/blob/master/app/providers/frontend/statistics_announcement_provider.rb

**Comment by boffbowsh** [comment]
Fair enough 👍

**Comment by tijmenb** [comment]
Added a comment to explain its purpose.

**Comment by boffbowsh** [comment]
👍

**Comment by tijmenb** [comment]
A comment that broke the build. Lol whitehall.

**Pull request merged**